gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=0.10.1
+version=0.0.0-development
```

Comment:
When Kafka is on version `3.8.1` (the next version is 3.9), they do: `3.9.0-SNAPSHOT`. Wondering if we can do that?

For our case, right now it would be `0.11.0-SNAPSHOT` since we're about to release `0.11.0`. But then we'd have to update it either manually or in the release pipeline, which might be too much work.